wallet/wallet.go

```diff
@@ -2975,6 +2976,32 @@ func (w *Wallet) StakeInfo() (*StakeInfoData, error) {
 			}
 		}
 
+		expiredTicketNum := 0
+		revokeTicketPtrs := make([]*chainhash.Hash, len(revokedTickets))
+		for i := range revokedTickets {
+			revokeTicketPtrs[i] = &revokedTickets[i]
+		}
+
+		// Check the expired ticket pool for the presense of tickets.
+		expiredBitSetBStr, err := chainClient.ExistsExpiredTickets(revokeTicketPtrs)
```

Comment:
Then does [`chainClient.MissedTickets()`](https://github.com/alexlyp/btcwallet/blob/c60d4e2133d4f02d581d9903219cdd6356fa928c/wallet/wallet.go#L3031) no longer include expired? Or does that count need to be decremented?